python/doc/usecases/use_case_stiffened_panel.rst

```diff
@@ -0,0 +1,207 @@
+.. _use-case-stiffened-panel:
+
+Stiffened panel buckling
+========================
+
+
+Introduction
+------------
+
+The following figure presents a stiffed panel subject to buckling on a military aircraft.
+
+This use-case implements a simplified model of buckling for a stiffened panel, detailed in [ko1994]_.
+
+.. figure:: ../_static/stiffened_panel_illustration.jpg
+    :align: center
+    :alt: buckling illustration
+    :width: 100%
+
+    **Figure 1.** Buckling of a stiffened panel.
+
+
+.. figure:: ../_static/stiffened_panel_simulation.png
+    :align: center
+    :alt: buckling simulation
+    :width: 100%
+
+    **Figure 2.** 3D simulation of buckling.
+
+
+.. figure:: ../_static/stiffened_panel_description.png
+    :align: center
+    :alt: stiffened panel geometry
+    :width: 100%
+
+    **Figure 3.** Parameterization of the stiffened panel.
+
+
+This test case is composed of nine random variables:
+
+- :math:`E\sim\mathcal{TN}(110.0e9, 55.0e9, 99.0e9, 121.0e9)` : Young modulus (Pa)
```

Comment:
Using $10e9$ has no particular mathematical meaning because there is no variable $e$ involved here. The mathematical value is $10^9$, of course.
```suggestion
- :math:`E\sim\mathcal{TN}(110.0 \times 10^{9}, 55.0e9, 99.0e9, 121.0e9)` : Young modulus (Pa)
```
Please fix the other ones similarily.

Reply:
I imitated the way the excellent Fire Satellite use case was written. I will fix it accordingly.

Reply:
I used the siunitx package to render the numerical values and the units, IMO it should be the way to go now.